### PR TITLE
Feat/Next-Frontend APIs

### PR DESCRIPTION
**Pull request by paradoxe35**
- POST /auth/send-code
- POST /auth/signin-email
- POST /auth/verify/code
- POST /timer/sync
- POST /auth/signin-email-confirm
- POST /auth/verify/token
- POST /auth/refresh

### CODE REVIEW

**Comment by paradoxe35** [comment]
@evereq 
You can review this PR

File: apps/web/app/hooks/auth/useAuthenticationPasscode.ts [hunk 1]
```diff
@@ -98,13 +100,17 @@ export function useAuthenticationPasscode() {
 					}
 				}
 
-				if (res.data?.status !== 200 && res.data?.status !== 201) {
-					setErrors({ code: t('pages.auth.INVALID_INVITE_CODE_MESSAGE') });
-				}
+				// if (res.data?.status !== 200 && res.data?.status !== 201) {
```
Comment (evereq) [inline apps/web/app/hooks/auth/useAuthenticationPasscode.ts:103]: @paradoxe35 why this commented out now?

Comment (paradoxe35) [reply apps/web/app/hooks/auth/useAuthenticationPasscode.ts:103]: Let me uncomment it
even though the code is no longer accurate.

Comment (paradoxe35) [reply apps/web/app/hooks/auth/useAuthenticationPasscode.ts:103]: Before the server could return 200 response even if it's a bad request. I've fixed

**Pull request merged**